<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsCarsPropViewerClass</class>
 <widget class="QMainWindow" name="QtWidgetsCarsPropViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsCarsPropViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_AddCar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>0</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_AddCar">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавити</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>49</y>
       <width>67</width>
       <height>191</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Марка</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Модель</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Рік випуску</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Колір</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Ціна</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Рег. номер</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>135</width>
       <height>192</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_id"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_mark"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_model"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_year"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_color"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_price"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_RegNum"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_Clear">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистити</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_ShowCars">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="tableCars">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>741</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>7</number>
     </property>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="pushButton_ApplyFilter">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>370</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Применити</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_C">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>260</y>
       <width>251</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_year_filter">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Рік</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ціна</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price_filter">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_B">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Модель</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_model_filter">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_year_more_filter">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Рік старіше</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_A">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>221</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Марка</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mark_filter">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>tableCars</zorder>
    <zorder>pushButton_ApplyFilter</zorder>
    <zorder>frame_B</zorder>
    <zorder>frame_C</zorder>
    <zorder>frame_A</zorder>
   </widget>
   <zorder>frame_ShowCars</zorder>
   <zorder>frame_AddCar</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Авто</string>
    </property>
    <addaction name="AddNewCar"/>
    <addaction name="action_A"/>
    <addaction name="action_B"/>
    <addaction name="action_C"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="AddNewCar">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="action_A">
   <property name="text">
    <string>Показать A</string>
   </property>
  </action>
  <action name="action_B">
   <property name="text">
    <string>Показать B</string>
   </property>
  </action>
  <action name="action_C">
   <property name="text">
    <string>Показать C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgetsCarsPropViewer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AddNewCar</sender>
   <signal>triggered()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>show_frame_add_car()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_AddCar</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>add_new_car()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_A</sender>
   <signal>triggered()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>show_frame_viewer_filter_mark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>419</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Clear</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>clear_car_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>775</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_B</sender>
   <signal>triggered()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>show_frame_viewer_filter_model_price()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_C</sender>
   <signal>triggered()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>show_frame_viewer_filter_year_price()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ApplyFilter</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsCarsPropViewerClass</receiver>
   <slot>apply_selected_filter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add_new_car()</slot>
  <slot>show_frame_add_car()</slot>
  <slot>show_frame_viewer_filter_year_price()</slot>
  <slot>clear_car_data()</slot>
  <slot>show_frame_viewer_filter_mark()</slot>
  <slot>show_frame_viewer_filter_model_price()</slot>
  <slot>apply_selected_filter()</slot>
 </slots>
</ui>
